charts/spin-operator/README.md

```diff
@@ -42,34 +42,31 @@ After installing the chart, you'll need to ensure the following:
 
 - An application executor is installed. This is the executor that spin-operator uses to run Spin applications.
 
-  <!-- TODO: templatize with release version corresponding to chart's appVersion -->
-
   ```console
-  $ kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.1.0-rc.1/spin-operator.shim-executor.yaml
+  $ kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/{{ APP_VERSION }}/spin-operator.shim-executor.yaml
   ```
 
 - A RuntimeClass resource for the `wasmtime-spin-v2` container runtime is installed. This is the runtime that Spin applications use.
 
-  <!-- TODO: templatize with release version corresponding to chart's appVersion -->
-
   ```console
-  $ kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.1.0-rc.1/spin-operator.runtime-class.yaml
+  $ kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/{{ APP_VERSION }}/spin-operator.runtime-class.yaml
```

Comment:
these might be better coming from config/samples (via http) - coming from a release artifact implies they are "valid production resources" to me - which they are explicitly not 😅

Reply:
yea I think it might be nice to keep paths to files on main and then for releasing, maybe we rely on sed without the templating?

Reply:
Made some updates in https://github.com/spinkube/spin-operator/pull/155/commits/d64faa3d741fa595d2771137f537940d2accc75d.  A bit awkward what with converting multiple CRD GH urls to the one combined yaml url attached to a release... and now I realize for all of the pre-release charts we publish from main, we'll be pointing to non-existent GH releases in this README.

Taking a step back, I wonder if we just replace most of the instructions in this README with a link to https://www.spinkube.dev/docs/spin-operator/installation/installing-with-helm/ and do away with the maintenance burden and duplication of instructions?  WDYT?